bindings/wasm/src/error.rs

```diff
@@ -211,9 +211,15 @@ impl JsValueResult {
   /// Consumes the struct and returns a Result<_, AccountStorageError>
   pub fn account_err(self) -> StdResult<JsValue, AccountStorageError> {
```

Comment:
From the name `account_err` I would expect this to return `Option<AccountStorageError>` since `std::Result<T,E>::error` returns `Option<E>`. Would it be possible to just inline this method in the `From<JsValueResult>` for  `AccountStorageResult` implementations if that is its intended use case?

Reply:
I wouldn't really want to inline the method into the two implementations, since that duplicates code.
How about renaming the method to `to_account_error`?

Reply:
`to_account_error` is better. If it is necessary for this to be a public method, then please also update the documentation to explain that the `Ok` variant is left untouched.